torch/ao/quantization/observer.py

```diff
@@ -330,7 +338,7 @@ def _calculate_qparams(
             max_val_pos = torch.max(-min_val_neg, max_val_pos)
             scale = max_val_pos / (float(quant_max - quant_min) / 2)
             scale = torch.max(scale, self.eps)
-            if self.dtype == torch.quint8:
+            if self.dtype in [torch.quint8, torch.uint8]:
```

Comment:
why did this have to be changed?

Reply:
this is to make sure the code works for both quint8 and uint8 dtypes